subprojects/docs/src/docs/release/notes.md

```diff
@@ -1,6 +1,16 @@
 The Gradle team is excited to announce Gradle @version@.
 
-This release features [1](), [2](), ... [n](), and more.
+This release features the support for [persistent Java compiler daemons](#faster-java-compilation) to speed up Java compilation.
+Gradle will also use [less memory for dependency resolution](#reduced-memory-consumption).
+The effect is significant for large builds, in particular, large Android builds including, but not limited to, IDE sync.
```

Comment:
```suggestion
The effect is significant, particularly for large builds such as Android builds.
```

Reply:
I don't see a way to add "IDE sync". It's just a random add at this point.

Reply:
Thanks, we have the specifics in the section. Making it shorter for the preamble makes sense IMO